numpy/linalg/setup.py

```diff
@@ -39,7 +39,12 @@ def configuration(parent_package='', top_path=None):
         class numpy_linalg_lapack_lite(system_info):
             def calc_info(self):
                 info = {'language': 'c'}
-                if sysconfig.get_config_var("SIZEOF_SIZE_T") >= 8:
+                size_t_size = sysconfig.get_config_var("SIZEOF_SIZE_T")
+                if size_t_size:
+                    maxsize = 2**(size_t_size - 1) - 1
+                else: # handle windows
```

Comment:
```suggestion
                else:
                    # We prefer using sysconfig as it allows cross-compilation
                    # but the information may be missing (e.g. on windows).
```
Just thought it might be nice to explain why we do this dance?